kitsune/settings.py

```diff
@@ -983,26 +983,26 @@ def show_toolbar_callback(*args):
 
 # CORS Setup
 CORS_ALLOW_ALL_ORIGINS = True
-CORS_URLS_REGEX = [
```

Comment:
Is this change needed for mypy?

Reply:
Yes, this was because `mypy` inferred that it was a list initially, and then complained when we tried to turn it into an instance of `re.Pattern`